src/dsp_tools/commands/excel2json/project.py

```diff
@@ -22,12 +22,13 @@ def excel2json(
     ::
 
         data_model_files
-        |-- lists
-        |   |-- de.xlsx
-        |   `-- en.xlsx
-        `-- onto_name (onto_label)
-            |-- properties.xlsx
-            `-- resources.xlsx
+        ├── json_header.xlsx (optional)
+        ├── lists
+        │   ├── list.xlsx
+        │   └── list_1.xlsx
```

Comment:
I think this is the old excel2json, so it should be like this:

```suggestion
        │   ├── de.xlsx
        │   └── en.xlsx
```